### ISSUE TITLE
torch::save

### ISSUE THREAD

**Issue by rmx564354093** [pytorch/pytorch#59613]
**In pytorch  when train model  over   we can save model .  such as** 
torch.save({
            'epoch': epoch,
            'model_state_dict': model.state_dict(),
            'optimizer_state_dict': optimizer.state_dict(),
            'loss': loss,
            ...
            }, PATH)

**and  load  model and param  such as**
 model.load_state_dict(checkpoint['model_state_dict'])
optimizer.load_state_dict(checkpoint['optimizer_state_dict'])
epoch = checkpoint['epoch']
loss = checkpoint['loss']

but  in libtorch .  I don't know how to achieve it .   Please help me?

**Comment by skyline75489** [comment]
Hi! Github is mainly for issues about PyTorch *itself*. This is a better fit for the [PyTorch forums ](https://discuss.pytorch.org/) so I'd suggest posting there as it's more likely you will get assistance.
